### ISSUE TITLE
[Dungeon] Stonecore 

### ISSUE THREAD

**Issue by MrTide** [gamefreedomgit/Maelstrom#1883]
1. stonecore earthshaper is not casting anything after respawning it one more time it did but looks like it has some scripting issues
![image](https://user-images.githubusercontent.com/36106057/212428683-aa417b0b-0a10-4114-80b4-69b0b37d5528.png)
2.  rock borers are not patroling the area 
![image](https://user-images.githubusercontent.com/36106057/212429435-088664ca-c0c0-45d6-91e3-f75a315f459f.png)
3. stonecore flayer does not do enough damage with flay ability and it should not follow player around and does it on one spot
https://cata-twinhead.twinstar.cz/?spell=79922
4. event just before slabhide is missing it should throw stones to the players and such more details on the video
https://youtu.be/Pg1L8uFYpzg?t=769
5. on slabhide rocks should be smaller and spread around more and also they are not dropping down properly, also crystal storm is not scripted properly its explained in the video you should hide behind the rocks to avoid it there is no possibility it to dodge it atm specially for the tanks since boss follows you
https://www.youtube.com/watch?v=J9NPfULwG5Q
6. https://github.com/gamefreedomgit/Maelstrom/issues/1868 missing spells also some chain agrro issues regarding the mobs in the same area 
7. stonecore magmalord should not be immune to banish
8.  ozruk he didnt enrage at 20% hp (removeable enrage) : https://www.youtube.com/watch?v=TdN2c4PHWB8
according to players his autoattack damage seems way too low his damage can be seen in this video https://www.youtube.com/watch?v=53G731ns514
![image](https://user-images.githubusercontent.com/36106057/212433494-a315ea91-68f6-4df1-8c39-31e7c20821d6.png)
9. gravity well missing (heroic version is different)
![image](https://user-images.githubusercontent.com/36106057/212434163-97266c49-1b60-464e-8742-dfb5d6abef99.png)
force grip was not working properly not interruptible,  https://www.wowhead.com/spell=79351/force-grip#comments:id=1248171
rocks animations are wrong they are too fast
ads got stuck on spawn
full fight here https://www.youtube.com/watch?v=y7GFLI_PCUM
9. devout follower in this dungeon keep dropping rogue pick pocket loot
 [Flame-Scarred Junkbox]  [Rogue's Draught]  [Embersilk Cloth] and so on
 basically they should not drop anything without pick pocket
 ![image](https://user-images.githubusercontent.com/36106057/212435518-87ed1d83-cfaa-4c01-851b-23545d14416b.png)

**Comment by Afgann** [comment]
fixed

**Comment by Hotandspicy322** [comment]
Slabhide prefight event :
The stones are not coming one by one as they should
they are coming in 2 batches (should not)

All of the Falling rocks (the ones in the bossfight and in the preevent)do exactly 0(zero) damage
they should do damage

Slabhide:
Hitbox too small

Stonecore Sentry:
Does not Patrol, none of them

Stonecore Bruiser:
missing charge attack

Ozruk:
too low damage on Groundslam (about 35k rn)

**Comment by WhitemaneSylah** [comment]
Issues not fixed,  See above

**Comment by WhitemaneSylah** [comment]
Test 18.01 New bugs round
Corborus THRASHING CRASH does not do damage across whole line 
https://www.wowhead.com/npc=43438/corborus

Slabhide - Pre-event  not dropping enough stalactites ,  should drop several

Slabhide fight - fixed no more bugs

Ozruk - fixed no more bugs

High Priestess Azil - new bugs found
https://wowpedia.fandom.com/wiki/High_Priestess_Azil

Force grip doesn't ever get used
Gravity well doesn't ever get used
Devout followers get stuck on 2nd wave and do not run in
Seismic shard does 0 damage
Transition to stage 2 is extremely late, should happen  at roughly 50%

**Comment by WhitemaneSylah** [comment]
![image](https://user-images.githubusercontent.com/121137397/219800402-ddda15c7-beae-478c-986e-3728db63ca4e.png)

Players are being stuck in combat after Staghide from the guids above

**Comment by Commander-Rug090** [comment]
**Corborus - Burrow**
He does two Thrashing charge
then disappear
the boss gets undermap - still not fixed.

**Comment by WhitemaneSylah** [comment]
#2243

**Comment by UltraNixx** [comment]
@WhitemaneSylah Can be closed this one?
